src/main/webapp/js/instructorHome.es6

```diff
@@ -52,15 +54,16 @@ $(document).ready(() => {
     });
 
     // AJAX loading of course panels
-    const $coursePanels = $('div[id|="course"]');
+    const $coursePanels = $('.ajax_auto');
     $.each($coursePanels, function () {
         $(this).filter(function () {
-            const isNotLoaded = $(this).find('form').length;
+            const isNotLoaded = $(this).parent().find('form').length;
             return isNotLoaded;
         }).click(function () {
             const $panel = $(this);
-            const formData = $panel.find('form').serialize();
+            const formData = $panel.parent().find('form').serialize();
             const content = $panel.find('.pull-right')[0];
+            const panelCollapse = $(this).parent().children('.panel-collapse');
```

Comment:
As a general rule, we prefix variable names with `$` if they refer to jQuery objects (such as this one). There are a few similar variables without the prefix in this file.